manager/runner.go

```diff
@@ -1478,3 +1511,30 @@ func newWatcher(c *config.Config, clients *dep.ClientSet) *watch.Watcher {
 		RetryFuncNomad:   watch.RetryFunc(c.Nomad.Retry.RetryFunc()),
 	})
 }
+
+func recordDependencyCounts(deps map[string]dep.Dependency) {
```

Comment:
this function doesn't return anything or modify the input in any way, what is it meant to do?